x-pack/plugins/cloud_security_posture/server/create_indices/create_processor.ts

```diff
@@ -25,26 +30,8 @@ export const createPipelineIfNotExists = async (
     const exitError = transformError(exitErr);
     if (exitError.statusCode === 404) {
       try {
-        await esClient.ingest.putPipeline({
```

Comment:
This code shouldn't be deleted, it used by createBenchmarkScoreIndex. 
https://github.com/elastic/kibana/pull/141095/files#diff-f86802a29feadf8c5b8336447ba954343c4b404dcab0785c66ee3a44149f5b78R49

we need to create additional pipeline, not replace this one.

Reply:
I am not sure I am following, the `esClient` still creates pipeline as it did before

Reply:
ignore it, I didn't meant to post it